applications/crossbar/src/cb_apps_util.erl

```diff
@@ -74,6 +75,41 @@ is_authorized(AccountDoc, UserId, App) ->
 %%%===================================================================
 %%% Internal functions
 %%%===================================================================
+-spec filter_apps(ne_binary(), wh_json:objects()) -> wh_json:objects().
+filter_apps(AccountId, Apps) ->
+    lists:foldl(
+        fun(App, Acc) ->
+            Name = wh_json:get_value(<<"name">>, App),
+            Filtered = filter_app(AccountId, App, Name),
+            case wh_json:is_empty(Filtered) of
+                'true' -> Acc;
+                'false' -> [Filtered|Acc]
+            end
+        end
+        ,[]
+        ,Apps
+    ).
+
+-spec filter_app(ne_binary(), wh_json:object(), ne_binary()) -> wh_json:object().
+filter_app(AccountId, App, <<"port">>) ->
+    lager:debug("filtering port application"),
+    ResellerId = wh_services:find_reseller_id(AccountId),
+    MaybeHide =
+        case kz_whitelabel:fetch(ResellerId) of
+            {'error', _R} ->
+                lager:error("failed to load whitelabel doc for ~s: ~p", [ResellerId, _R]),
+                'false';
```

Comment:
Shouldn't we hide it, as a precaution, on errors? Maybe show it on `{'error', 'not_found'}` but otherwise hide it?